docs/design/deployment-tool.md

```diff
@@ -0,0 +1,173 @@
+# AWS .NET Deployment Tool Design Doc
+
+The purpose of this document is to describe the high level components of the new .NET Deployment Tool that is currently being developed by the AWS .Net Developer Experience team. The .NET Deployment Tool aims to improve the development and deployment of .NET cloud-native applications on AWS. .NET developers can deploy their applications in just a few steps from the .NET CLI and accelerate application development by using use-case specific libraries and application templates.
+\
+&nbsp;
+
+## Problem Statement
+
+When .NET developers begin their cloud journey, many of them try to deploy an existing application on AWS, and are often overwhelmed by the abundance of AWS service offerings. They struggle to select the right compute service, provision the underlying infrastructure, and understand how to build, package, and deploy their application artifacts. Often times, they give up during the early stages of the process. 
+
+As developers progress to the adoption phase, many of them prefer creating .NET microservices using containers or Lambda to minimize the infrastructure management. They have to use low level service APIs, which often leads to more boilerplate code than business logic code. Additionally, this process needs to be repeated by every developer building a similar application, increasing the odds of inconsistencies across applications. Furthermore, there is no way to collaborate across teams or the organization to develop and deploy to AWS.
+\
+&nbsp;
+
+## Proposal
+
+The experience that will be created with this tool will allow .NET developers to take their .NET cloud-ready application that they got started in their IDE of choice and easily deploy it to AWS while learning about AWS deployment services and what the appropriate service for their application is.
+
+_We want to onboard developers to AWS quickly, so that they realize the benefits of AWS without spending hours reading documentation._ said Ken Exner, Director, AWS Developer Tools. The new features and improved tools allow developers to make the right choice of compute for their .NET applications, and to rapidly deploy their .NET applications using AWS best practices. They also greatly improve productivity by reducing the amount of non-differentiating code developers need to write and maintain for their applications.
```

Comment:
This paragraph is all marketing and doesn't need to be in a tech design document.

Reply:
Based on the ticket that irene created https://sim.amazon.com/issues/DOTNET-5133 we wanted the doc to have a problem statement and use cases (which is why the top part looks like a marketing document). How do you propose we move forward?

Reply:
I removed that block.